kie-wb-tests/kie-wb-tests-rest/src/main/java/org/kie/wb/test/rest/client/RestWorkbenchClient.java

```diff
@@ -204,29 +192,17 @@ public CreateProjectRequest createProject(String repositoryName, String projectN
     }
 
     @Override
-    public CreateProjectRequest createProject(String repositoryName, ProjectRequest project) {
-        log.info("Creating project '{}' in repository '{}'", project.getName(), repositoryName);
+    public CreateProjectRequest createProject(String organizationalUnitName, ProjectRequest project) {
+        log.info("Creating project '{}' in repository '{}'", project.getName(), organizationalUnitName);
 
-        CreateProjectRequest request = target.path("repositories/{repositoryName}/projects")
-                .resolveTemplate("repositoryName", repositoryName)
+        CreateProjectRequest request = target.path("organizationalUnits/{organizationalUnitName}/projects")
+                .resolveTemplate("organizationalUnitName", organizationalUnitName)
                 .request().post(createEntity(project), CreateProjectRequest.class);
 
         return waitUntilJobFinished(request, projectJobTimeoutSeconds);
     }
 
     @Override
-    public DeleteProjectRequest deleteProject(String repositoryName, String projectName) {
-        log.info("Removing project '{}' from repository '{}'", projectName, repositoryName);
-
-        DeleteProjectRequest request = target.path("repositories/{repositoryName}/projects/{projectName}")
-                .resolveTemplate("repositoryName", repositoryName)
-                .resolveTemplate("projectName", projectName)
-                .request().delete(DeleteProjectRequest.class);
-
-        return waitUntilJobFinished(request, projectJobTimeoutSeconds);
-    }
-
-    @Override
     public Collection<ProjectResponse> getProjects(String repositoryName) {
```

Comment:
This method is not updated. In the return is used path "repositories/{repositoryName}/projects".